src/redux/Provider.tsx

```diff
@@ -49,7 +49,10 @@ const ReduxProvider = ({ children, locale }) => {
         const reciterId =
           userPreferences[PreferenceGroup.AUDIO]?.reciter ||
           audioService.getSnapshot().context.reciterId;
-        audioService.send({ type: 'SET_INITIAL_CONTEXT', playbackRate, reciterId });
+        const volume =
+          userPreferences[PreferenceGroup.AUDIO]?.volume ||
```

Comment:
I don't `userPreferences[PreferenceGroup.AUDIO]?.volume` is necessary since we are not storing the value of `volume` remotely.